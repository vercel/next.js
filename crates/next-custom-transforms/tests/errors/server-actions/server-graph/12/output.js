/* __next_internal_client_entry_do_not_use__ foo auto */ const { createProxy } = require("private-next-rsc-mod-ref-proxy");
module.exports = createProxy("/app/item.js");
