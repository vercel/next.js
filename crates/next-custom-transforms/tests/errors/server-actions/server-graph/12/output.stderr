  x It's not possible to have both `use client` and `use server` directives in the same file.
   ,-[input.js:2:1]
 1 | 'use server'
 2 | 'use client'
   : ^^^^^^^^^^^^
   `----
